tbx/static_src/sass/config/_mixins.scss

```diff
@@ -82,21 +82,91 @@
     Font sizes
 */
 @mixin font-size($keyword) {
-    $small-size: map.get($small-font-sizes, $keyword);
-    $medium-size: map.get($medium-font-sizes, $keyword);
-    $large-size: map.get($large-font-sizes, $keyword);
+    // Assuming each value in the font-size maps is a list of (size, line-height)
```

Comment:
Added the line height to this mixin and in the sass ma[p in the variables file. Felt cleaner rather than setting them on the `.heading` class.